### DISCUSSION TITLE
Annoying snapping

### DISCUSSION THREAD

**Discussion by mrYodak** [godotengine/godot-proposals#8859] [3D]
When i work on a 3d scene and placing assets, often they would start snapping to a grid even without the snap toggle. And the snap toggle actually makes them not snap and move freely. But then they would start snapping again, which is fixed by turning off the snap toggle. And so it goes... It's not a big deal, but its annoying to deal with.

**Comment by Calinou** [comment]
This is a bug: https://github.com/godotengine/godot/issues/83777
